src/core/channel/client_setup.c

```diff
@@ -148,7 +180,8 @@ int grpc_client_setup_cb_begin(grpc_client_setup_request *r) {
   return 1;
 }
 
-void grpc_client_setup_cb_end(grpc_client_setup_request *r) {
+void grpc_client_setup_cb_end(grpc_client_setup_request *r,
+                              const char *reason) {
```

Comment:
Just checking: is "reason" meant for future use? It not currently referenced anywhere in the function body.